geps/gep-713/index.md

```diff
@@ -3,1155 +3,1172 @@
 
-Ana sighs.
+Example of Effective metaresources based on a hierarchy of Gateway API resources:
 
-**Ana**: _Wish I could. Wind’s died down, though, and it'll be dark soon.
-Just gonna head home._
+```mermaid
+---
+config:
+  look: handDrawn
+  theme: neutral
+---
+graph
+    gc@{ shape: rect, label: "GatewayClass 1" }
+    g@{ shape: rect, label: "Gateway 1" }
+    r1@{ shape: rect, label: "Route 1" }
+    r2@{ shape: rect, label: "Route 2" }
+    b@{ shape: rect, label: "Backend 1" }
+    m1@{ shape: stadium, label: "Metaresource 1" }
+    m2@{ shape: stadium, label: "Metaresource 2" }
 
-**Chihiro**: _Ouch. Sorry to hear that._ 😐
+    gc --> g
+    g --> r1
+    g --> r2
+    r1 --> b
+    r2 --> b
 
-One more look out at the lake.
+    m1 -.-> g
+    m2 -.-> r1
+```
 
-**Ana**: _Thanks for the help. Wish we’d found better answers._ 😢
+The above yields 2 Effective metaresources:
+- For `Route 1`: a combination of `Metaresource 1` and `Metaresource 2`
+- For `Route 2`: equal to `Metaresource 1`
 
-### The Problem, restated
-What this parable makes clear is that, in the absence of information about what
-Policy is affecting an object, it’s very easy to make poor decisions.
+#### Merge strategies
 
-It’s critical that this proposal solve the problem of showing up to three things,
-listed in increasing order of desirability:
+If multiple metaresources have the same scope (that is, multiple CRs based on the same metaresource kind affect the same effective target), this is considered to be a _conflict_.
 
-- _That_ some Policy is affecting a particular object
-- _Which_ Policy is (or Policies are) affecting a particular object
-- _What_ settings in the Policy are affecting the object.
+Conflicts MUST be resolved according to some _merge strategy_. A merge strategy is a function that receives two conflicting specs and returns a new spec without conflict.
 
-In the parable, if Ana and Chihiro had known that there were Policies affecting
-the relevant object, then they could have gone looking for the relevant Policies
-and things would have played out differently. If they knew which Policies, they
-would need to look less hard, and if they knew what the settings being applied
-were, then the parable would have been able to be very short indeed.
+##### Basic merge strategies
 
-(There’s also another use case to consider, in that Chihiro should have been able
-to see that the Policy on the namespace was in use in many places before deleting
-it.)
+3 *basic merge strategies* are defined:
```

Comment:
"merge strategy" seems confusing here as the term should be reserved for how 2 policies are merged, and not to determine their relative prioritization on who decides how they are merged. None/Defaults/Overrides should be categorized as "policy priority". 

The atomicity levels defined below reflect the actual merge strategy.